tests/model/generated/object/test_notification_url.py

```diff
@@ -0,0 +1,217 @@
+    JSON_PATH_MONETARY_ACCOUNT_BANK_MODEL = \
+        BASE_PATH_JSON_MODEL + '/MonetaryAccountBank.json'
+    JSON_PATH_PAYMENT_BATCH_MODEL = \
+        BASE_PATH_JSON_MODEL + '/PaymentBatch.json'
+    JSON_PATH_REQUEST_INQUIRY_MODEL = \
+        BASE_PATH_JSON_MODEL + '/RequestInquiry.json'
+    JSON_PATH_REQUEST_RESPONSE_MODEL = \
+        BASE_PATH_JSON_MODEL + '/RequestResponse.json'
+    JSON_PATH_SCHEDULE_PAYMENT_MODEL = \
+        BASE_PATH_JSON_MODEL + '/ScheduledPayment.json'
+    JSON_PATH_SCHEDULE_INSTANCE_MODEL = \
+        BASE_PATH_JSON_MODEL + '/ScheduledInstance.json'
+    JSON_PATH_SHARE_INVITE_BANK_INQUIRY_MODEL = \
+        BASE_PATH_JSON_MODEL + '/ShareInviteBankInquiry.json'
+    JSON_PATH_SHARE_INVITE_BANK_RESPONSE_MODEL = \
+        BASE_PATH_JSON_MODEL + '/ShareInviteBankResponse.json'
+
+    # Model root key.
+    _KEY_NOTIFICATION_URL_MODEL = 'NotificationUrl'
+
+    def execute_notification_url_test(self,
+                                      file_path,
+                                      class_name,
+                                      getter_name):
+        """
+        :type file_path: str
+        :type class_name: str
+        :type getter_name: str
+
+        :return: None
+        """
+
+        notification_url = self.getNotificationUrl(file_path)
+        self.assertIsNotNone(notification_url)
+        self.assertIsNotNone(notification_url.object_)
+
+        expected_model = getattr(notification_url.object_, getter_name)
+        referenced_model = notification_url.object_.get_referenced_object()
+
+        self.assertIsNotNone(expected_model)
+        self.assertIsNotNone(referenced_model)
+        self.assertTrue(
+            self.assertInstanceOfReferencedObject(
+                referenced_model,
+                class_name
+            )
+            or
+            self.assertInstanceOfReferencedEndpoint(
+                referenced_model,
+                class_name
+            )
+        )
+
+    @staticmethod
+    def assertInstanceOfReferencedObject(referenced_model, class_name):
+        try:
+            return isinstance(referenced_model, getattr(object_, class_name))
+        except AttributeError:
+            return False
+
+    @staticmethod
+    def assertInstanceOfReferencedEndpoint(referenced_model, class_name):
+        try:
+            return isinstance(referenced_model, getattr(endpoint, class_name))
+        except AttributeError:
+            return False
+
+    def getNotificationUrl(self, file_path):
+        """
+        :type file_path: str
+
+        :rtype: object_.NotificationUrl
+        """
+
+        base_path = os.path.dirname(__file__)
+        file_path = os.path.abspath(os.path.join(base_path, file_path))
+
+        with open(file_path, 'r') as f:
+            json_string = f.read()
+            json_object = json.loads(json_string)
+            json_string = json.dumps(json_object[
+                                         self._KEY_NOTIFICATION_URL_MODEL
```

Comment:
since we've started... %)
here I'd prefer to have it formatted like this:
```
json_string = json.dumps(
    json_object[self._KEY_NOTIFICATION_URL_MODEL]
)
```

Because it is an object element access rather than an array definition, and multi-line approach seems highly odd here.